MediaBrowser.Controller/SyncPlay/IGroupController.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Threading;
+using Jellyfin.Data.Entities;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Controller.SyncPlay.Queue;
+using MediaBrowser.Controller.SyncPlay.Requests;
+using MediaBrowser.Model.SyncPlay;
+
+namespace MediaBrowser.Controller.SyncPlay
+{
+    /// <summary>
+    /// Interface IGroupController.
+    /// </summary>
+    public interface IGroupController
```

Comment:
Will there be more implementations of this? Otherwise I don't think there's any value in having it. As it is right now, GroupController could just be a `Group` class that is in charge of its own state, which the SyncPlayManager manipulates at will.

Reply:
I agree on this one as well, I followed the Java way of passing objects around but `GroupController` will be used internally only by `SyncPlayManager` so yeah, the interface can be dropped.